operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -181,6 +181,9 @@ type CassandraDatacenterSpec struct {
 
 	// Container image for the log tailing sidecar container.
 	SystemLoggerImage string `json:"systemLoggerImage,omitempty"`
+
+	// AdditionalServiceConfig allows to define additional parameters that are included in the created Services. Note, user can override values set by cass-operator and doing so could break cass-operator functionality.
```

Comment:
👍 , but we'll probably have to document which labels they should avoid, somewhere

Reply:
@burmanm can you document the labels that need to be avoided? I think there are only a handful, in which case maybe just list them in the comments here.